BraveWallet/Settings/WalletSettingsView.swift

```diff
@@ -61,6 +63,7 @@ public struct WalletSettingsView: View {
         message: Text(Strings.Wallet.settingsResetWalletAlertMessage),
         primaryButton: .destructive(Text(Strings.Wallet.settingsResetWalletAlertButtonTitle), action: {
           settingsStore.reset()
+          presentationMode.dismiss()
```

Comment:
Careful as this will not work in iOS 14, honestly it would be better if you handled the dismissal in the `SettingsViewController` keyring reset method